synapse/lib/lmdbslab.py

```diff
@@ -581,17 +588,21 @@ def size(self, name):
     def offset(self, name):
         return self.offsets.get(name)
 
-    async def add(self, name, info):
+    async def add(self, name, qdef):
+        s_schemas.reqValidQueueDef(qdef)
```

Comment:
Looks like MultiQueue is also used by search, swarm, and optic (and obviously jsonstor too). Are you planning on updating those services too? Should we even be checking the inputs against the cortex queue schema here?